pci_windows.go

```diff
@@ -0,0 +1,87 @@
+// Use and distribution licensed under the Apache license version 2.
+//
+// See the COPYING file in the root project directory for full text.
+//
```

Comment:
Let's leave PCI information on Windows to a separate pull request, OK? I need to do some more research on just how relevant the PnPEntity thing in WMI is to a PCI device. Please just pull all the PCI code out into a separate pull request.

Reply:
Right now GPU e PCI are linked to try to follow the same pattern used on linux.

I had the same doubts, but the PNPDeviceID was the only working way I could get the Vendor/Product device data about the GPU.

If you prefer I can split GPU and PCI leaving the PCI to posterity

Reply:
Yeah, let's split the GPU and PCI stuff for a separate PR where we can discuss this topic in isolation.